test/utils/session/client.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const jstp = require('../../..');
+
+const appName = 'testApp';
+const iface = 'iface';
+const method = 'method';
+
+let connection = null;
+const client = { session: null };
+
+const connect = (port) => {
+  jstp.net.connect(appName, null, port, (error, conn, session) => {
+    if (error) {
+      process.send(['error', error]);
+    }
+    connection = conn;
+    client.session = session;
+    connection.callMethod(iface, method, [], (error) => {
+      if (error) {
+        process.send(['error', error]);
+      }
+    });
+  });
+};
+
+const getSession = () => {
```

Comment:
This method sends serialized session to master process but is called getSession, should probably be renamed to sendSession.